inc/saiport.h

```diff
@@ -1566,6 +1547,14 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_PRBS_RX_STATUS,
 
+    /**
+     * @brief Attribute data for #SAI_PORT_ATTR_PRBS_RX_STATE
```

Comment:
add some comments here to explain why we introduce this attribute and not reuse existing attribute.

a simplified version of following para is good.

There are many “clear on read” states in hardware. Sometimes these are individual standalone fields. "Many a times HW combines multiple fields in a single atomic read. 
Given that SAI API attributes are defined at sub field granularity, SAI adapter has to emulate “clear on read” for sub field queries. Flex state complex data type allows read operation on the combination of such attributes. This makes SAI adapter implementation easy by just making a single query to hardware. Hardware natively takes care of clear on read for all the subfields with in the field.

Reply:
https://github.com/opencomputeproject/SAI/pull/1179/commits/97ed2bb8e433f7ce80e9e898c6c75c4d8eff9d06

Done